tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderEntity.cs

```diff
@@ -1,99 +1,130 @@
+                Description,
+                OrganisationId,
+                OrganisationName,
+                OrganisationOdsCode,
+                OrganisationAddressId,
+                OrganisationBillingAddressId,
+                OrganisationContactId,
+                OrderStatusId,
+                Created,
+                LastUpdated,
+                LastUpdatedBy,
+                LastUpdatedByName
+            )
+            VALUES
+            (
+                @OrderId,
+                @Description,
+                @OrganisationId,
+                @OrganisationName,
+                @OrganisationOdsCode,
+                @OrganisationAddressId,
+                @OrganisationBillingAddressId,
+                @OrganisationContactId,
+                @OrderStatusId,
+                @Created,
+                @LastUpdated,
+                @LastUpdatedBy,
+                @LastUpdatedByName
             );";
-
+
         public static async Task<string> FetchOrderDescriptionFromOrderId(string connectionString, string orderId)
-        {
+        {
             return (await SqlRunner.QueryFirstAsync<string>(connectionString, @"SELECT
-                         [Description]
+                         [Description]
                          FROM dbo.[Order]
                          WHERE OrderId = @orderId;", new { orderId }));
-        }
-
-        public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
-        {
+        }
+
+        public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
+        {
             return (await SqlRunner.QueryFirstAsync<string>(connectionString, @"SELECT
                          LastUpdatedByName
                          FROM dbo.[Order]
                          WHERE OrderId = @orderId;", new { orderId }));
-        }
-
+        }
+
         public static async Task<OrderEntity> FetchOrderByOrderId(string connectionString, string orderId)
         {
             return (await SqlRunner.QueryFirstAsync<OrderEntity>(connectionString, @"SELECT
                           OrderId,
-                          [Description],
+                          [Description],
                           OrganisationId,
+                          OrganisationName,
+                          OrganisationOdsCode,
                           OrderStatusId,
                           Created,
                           LastUpdated,
                           LastUpdatedBy,
                           LastUpdatedByName
-                         FROM dbo.[Order]
-                         WHERE OrderId = @orderId;", new { orderId }));
-        }
-    }
-}
+                         FROM [Order]
+                         WHERE [OrderId] = @orderId", new { orderId }));
+        }
+
+        public static async Task<ContactEntity> FetchPrimaryContactByOrderId(string connectionString, string orderId)

```

Comment:
When you get to this give me a call, as I think I have an easier way of doing this, but its easier explained over call